calcSlope1 was not declared

@diizy, for some reason `stable-1.1` travis is failing with the following message:
```
/home/travis/build/LMMS/lmms/plugins/monstro/Monstro.cpp: In member function ‘void MonstroSynth::updateModulators(float*, float*, float*, float*, int)’:
/home/travis/build/LMMS/lmms/plugins/monstro/Monstro.cpp:787:56: error: ‘calcSlope1’ was not declared in this scope
/home/travis/build/LMMS/lmms/plugins/monstro/Monstro.cpp:797:70: error: ‘calcSlope1’ was not declared in this scope
/home/travis/build/LMMS/lmms/plugins/monstro/Monstro.cpp:811:63: error: ‘calcSlope1’ was not declared in this scope
/usr/bin/cmake -E cmake_progress_report /home/travis/build/LMMS/lmms/build/CMakeFiles 
```

Not sure if this is Travis upset or if it is an actual bug... The commit says it failed too... https://github.com/LMMS/lmms/commit/e33645f9aba8dc4e007dc02f94c8084bfc4eba74
https://travis-ci.org/LMMS/lmms/builds/47164992

--- a/plugins/monstro/Monstro.cpp
+++ b/plugins/monstro/Monstro.cpp
@@ -784,7 +784,7 @@ inline void MonstroSynth::updateModulators( float * env1, float * env2, float * 
 			}
 			else if( m_env_phase[i] < 2.0f ) // attack phase
 			{
-				env[i][f] = calcSlope1( fraction( m_env_phase[i] ) );
+				env[i][f] = calcSlope( i, fraction( m_env_phase[i] ) );
 				m_env_phase[i] = qMin( 2.0f, m_env_phase[i] + m_env_att[i] );
 			}
 			else if( m_env_phase[i] < 3.0f ) // hold phase
@@ -794,7 +794,7 @@ inline void MonstroSynth::updateModulators( float * env1, float * env2, float * 
 			}
 			else if( m_env_phase[i] < 4.0f ) // decay phase
 			{
-				const sample_t s = calcSlope1( 1.0f - fraction( m_env_phase[i] ) );
+				const sample_t s = calcSlope( i, 1.0f - fraction( m_env_phase[i] ) );
 				if( s <= m_env_sus[i] )
 				{
 					env[i][f] = m_env_sus[i];
@@ -808,7 +808,7 @@ inline void MonstroSynth::updateModulators( float * env1, float * env2, float * 
 			}
 			else if( m_env_phase[i] < 5.0f ) // release phase
 			{
-				env[i][f] = calcSlope1( 1.0f - fraction( m_env_phase[i] ) );
+				env[i][f] = calcSlope( i, 1.0f - fraction( m_env_phase[i] ) );
 				m_env_phase[i] += m_env_rel[i];
 			}
 			else env[i][f] = 0.0f;

